### ISSUE TITLE
Missing directory during build

### ISSUE THREAD

**Issue by danrl** [openwrt/packages#3515]
Maintainer: @zorun
Environment: all

Description: Missing directory during build.

```
...
make[3]: Leaving directory '/srv/OpenWRT/openwrt/package/libs/libmnl'
make[3]: Entering directory
'/srv/OpenWRT/openwrt/package/network/utils/resolveip'
rm -rf /srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/root-ar71xx/tmp-resolveip
mkdir -p /srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/root-ar71xx/stamp
/srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/root-ar71xx/tmp-resolveip
install -d -m0755
/srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/root-ar71xx/tmp-resolveip/usr/bin
install -m0755 /srv/OpenWRT/openwrt/build_dir/target-mips_34kc_musl-1.1.15/resolveip/resolveip
/srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/root-ar71xx/tmp-resolveip/usr/bin/
SHELL= flock /srv/OpenWRT/openwrt/tmp/.root-copy.flock -c 'cp -fpR
/srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/root-ar71xx/tmp-resolveip/.
/srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/root-ar71xx/'
rm -rf /srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/root-ar71xx/tmp-resolveip
touch /srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/root-ar71xx/stamp/.resolveip_installed
if [ -f /srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/pkginfo/resolveip.default.install.clean
]; then rm -f /srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/pkginfo/resolveip.default.install
/srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/pkginfo/resolveip.default.install.clean;
fi; echo "resolveip" >>
/srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/pkginfo/resolveip.default.install
make[3]: Leaving directory
'/srv/OpenWRT/openwrt/package/network/utils/resolveip'
make[3]: Entering directory '/srv/OpenWRT/openwrt/feeds/packages/net/wireguard'
mkdir -p /srv/OpenWRT/openwrt/bin/ar71xx/packages
/srv/OpenWRT/openwrt/build_dir/target-mips_34kc_musl-1.1.15/WireGuard-experimental-0.0.20161110/ipkg-ar71xx/wireguard-tools/CONTROL
/srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/pkginfo
install -d -m0755
/srv/OpenWRT/openwrt/build_dir/target-mips_34kc_musl-1.1.15/WireGuard-experimental-0.0.20161110/ipkg-ar71xx/wireguard-tools/usr/bin/
install -m0755 /srv/OpenWRT/openwrt/build_dir/target-mips_34kc_musl-1.1.15/WireGuard-experimental-0.0.20161110/src/tools/wg
/srv/OpenWRT/openwrt/build_dir/target-mips_34kc_musl-1.1.15/WireGuard-experimental-0.0.20161110/ipkg-ar71xx/wireguard-tools/usr/bin/
install -m0755 ./files/wireguard.sh
/srv/OpenWRT/openwrt/build_dir/target-mips_34kc_musl-1.1.15/WireGuard-experimental-0.0.20161110/ipkg-ar71xx/wireguard-tools/lib/netifd/proto/
install: target
'/srv/OpenWRT/openwrt/build_dir/target-mips_34kc_musl-1.1.15/WireGuard-experimental-0.0.20161110/ipkg-ar71xx/wireguard-tools/lib/netifd/proto/'
is not a directory: No such file or directory
Makefile:106: recipe for target
'/srv/OpenWRT/openwrt/bin/ar71xx/packages/packages/wireguard-tools_0.0.20161110-2_ar71xx.ipk'
failed
make[3]: *** [/srv/OpenWRT/openwrt/bin/ar71xx/packages/packages/wireguard-tools_0.0.20161110-2_ar71xx.ipk]
Error 1
make[3]: Leaving directory '/srv/OpenWRT/openwrt/feeds/packages/net/wireguard'
package/Makefile:196: recipe for target
'package/feeds/packages/wireguard/compile' failed
make[2]: *** [package/feeds/packages/wireguard/compile] Error 2
make[2]: Leaving directory '/srv/OpenWRT/openwrt'
package/Makefile:193: recipe for target
'/srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/stamp/.package_compile'
failed
make[1]: *** [/srv/OpenWRT/openwrt/staging_dir/target-mips_34kc_musl-1.1.15/stamp/.package_compile]
Error 2
make[1]: Leaving directory '/srv/OpenWRT/openwrt'
/srv/OpenWRT/openwrt/include/toplevel.mk:192: recipe for target 'world' failed
make: *** [world] Error 2
```

**Comment by danrl** [comment]
Fix in PR #3516

**Comment by danrl** [comment]
Fix merged. Closed.
